README.md

```diff
@@ -1,2 +1,104 @@
 # micromasters
 Portal for learners and course teams to access MITx micromasters programs
+
+Getting Started
+---------------
+
+You can either run micromasters locally with a default sqlite database after
+installing the ``requirements.txt`` file, or if you have Docker and
+prefer a cleaner environment, install ``docker-compose`` with ``pip
+install docker-compose`` and run ``docker-compose up``. This will set
+up a near production-ready containerized development environment that
+runs migrations, with the Django development server running on
+port ``8079``.
+
+To run one-off commands, like shell, you can run::
+
+  docker-compose run web python manage.py shell
+
+or to create root user::
+
+  docker-compose run web python manage.py createsuperuser
+
+
+For OS X Development
+--------------------
+
+Due to issues using Docker in OSX, development is more efficient if you
+install `docker-osx-dev <https://github.com/brikis98/docker-osx-dev>`_.
+``docker-osx-dev`` keeps host file system changes synced to the Docker
+container. You can install it by typing ``make`` once you have created the
+Docker container.
+
+Subsequently, before you start your Docker container with
+``docker-compose up``, you would run::
+  
+     docker-osx-dev -m default -s ./
```

Comment:
I think we'll want to change `default` to `mm` here and in the Makefile, but we can do this in a separate PR.

Can you add ` --ignore-file '.rsync-ignore'` to the end of this line?